rotkehlchen/tasks/calendar.py

```diff
@@ -47,3 +70,98 @@ def delete_past_calendar_entries(database: DBHandler) -> None:
             'DELETE FROM calendar WHERE timestamp < ? AND auto_delete=1',
             (ts_now(),),
         )
+
+
+def maybe_create_ens_reminders(database: DBHandler) -> None:
+    """Check ENS registration and renewal history events and create reminders if needed."""
+    db_calendar = DBCalendar(database=database)
+    db_history_events = DBHistoryEvents(database=database)
+    customizable_date = CustomizableDateMixin(database=database)
+    with database.conn.read_ctx() as cursor:
+        registrations: list['EvmEvent'] = db_history_events.get_history_events(
```

Comment:
is the type hint needed? You provided `EvmEventFilterQuery` that should be enough